--- hardware-handoff/spl_bsp/Makefile
+++ hardware-handoff/spl_bsp/Makefile
@@ -72,11 +72,9 @@ define stamp
 endef
 
 define untar_recipe
-$(UNTAR) $(if $1,$1,$(if $<,$<,$(error ERROR: no input provided to gnu make function untar_recipe)))
 endef
 
 # Stamps
-UNTAR_SRC := $(STAMP_DIR)/.untar
 CONFIG := $(STAMP_DIR)/.config
 PATCH_APPLY := $(STAMP_DIR)/.patch
 
@@ -211,7 +209,7 @@ tools: $(CONFIG)
 .PHONY: src
 src: $(UNTAR_SRC)
 
-$(UNTAR_SRC): $(TGZ)
+$(UNTAR_SRC):
 	@$(RM) $(PRELOADER_SRC_DIR)
 	$(untar_recipe)
 	@$(CHMOD) -R 755 $(PRELOADER_SRC_DIR)
@@ -237,7 +235,7 @@ $(UPDATE_GENERATED_SOURCE_FILES): $(PREL
 .PHONY: config
 config: $(CONFIG)
 
-$(CONFIG): $(UPDATE_SRC) $(UNTAR_SRC) $(PATCH_APPLY)
+$(CONFIG): $(UPDATE_SRC)
 	$(MAKE) $(MAKE_ARGS) -C $(PRELOADER_SRC_DIR) $(SOCFPGA_BOARD_CONFIG)
 	$(stamp)
 
