diff --git a/hardware-handoff/spl_bsp/Makefile b/hardware-handoff/spl_bsp/Makefile
index ec12f34..b741a77 100644
--- a/hardware-handoff/spl_bsp/Makefile
+++ b/hardware-handoff/spl_bsp/Makefile
@@ -72,11 +72,9 @@ define stamp
 endef
 
 define untar_recipe
-$(UNTAR) $(if $1,$1,$(if $<,$<,$(error ERROR: no input provided to gnu make function untar_recipe)))
 endef
 
 # Stamps
-UNTAR_SRC := $(STAMP_DIR)/.untar
 CONFIG := $(STAMP_DIR)/.config
 PATCH_APPLY := $(STAMP_DIR)/.patch
 
@@ -190,7 +188,7 @@ $(PRELOADER.BINARY): $(CONFIG)
 mkpimage-spl: $(PRELOADER.MKPIMAGE_BINARY)
 
 $(PRELOADER.MKPIMAGE_BINARY): $(PRELOADER.BINARY)
-	mkpimage --header-version 0 -o $@ $< $< $< $<
+	mkpimage -o $@ $< $< $< $<
 
 .PHONY: tools
 tools: $(CONFIG)
@@ -211,7 +209,7 @@ tools: $(CONFIG)
 .PHONY: src
 src: $(UNTAR_SRC)
 
-$(UNTAR_SRC): $(TGZ)
+$(UNTAR_SRC):
 	@$(RM) $(PRELOADER_SRC_DIR)
 	$(untar_recipe)
 	@$(CHMOD) -R 755 $(PRELOADER_SRC_DIR)
@@ -237,7 +235,7 @@ $(UPDATE_GENERATED_SOURCE_FILES): $(PRELOADER_UPDATE_DIR)/%: generated/% $(UNTAR
 .PHONY: config
 config: $(CONFIG)
 
-$(CONFIG): $(UPDATE_SRC) $(UNTAR_SRC) $(PATCH_APPLY)
+$(CONFIG): $(UPDATE_SRC)
 	$(MAKE) $(MAKE_ARGS) -C $(PRELOADER_SRC_DIR) $(SOCFPGA_BOARD_CONFIG)
 	$(stamp)
 
