diff --git a/hardware-handoff/spl_bsp/generated/sdram/sdram_config.h b/hardware-handoff/spl_bsp/generated/sdram/sdram_config.h
index 05d3ce2..5b45c4b 100644
--- a/hardware-handoff/spl_bsp/generated/sdram/sdram_config.h
+++ b/hardware-handoff/spl_bsp/generated/sdram/sdram_config.h
@@ -33,7 +33,7 @@
 #define CONFIG_HPS_SDR_CTRLCFG_CTRLCFG_MEMTYPE			(2)
 #define CONFIG_HPS_SDR_CTRLCFG_CTRLCFG_MEMBL			(8)
 #define CONFIG_HPS_SDR_CTRLCFG_CTRLCFG_ADDRORDER		(0)
-#define CONFIG_HPS_SDR_CTRLCFG_CTRLCFG_ECCEN			(1)
+#define CONFIG_HPS_SDR_CTRLCFG_CTRLCFG_ECCEN			(0)
 #define CONFIG_HPS_SDR_CTRLCFG_CTRLCFG_ECCCORREN		(1)
 #define CONFIG_HPS_SDR_CTRLCFG_CTRLCFG_REORDEREN		(1)
 #define CONFIG_HPS_SDR_CTRLCFG_CTRLCFG_STARVELIMIT		(10)
